deepmedic/frontEnd/configParsing/trainSessionParams.py

```diff
@@ -7,346 +7,455 @@
 
```

Comment:
Dont run + commit PEP in the same commit as actual changes. Makes it impossible to find them during code review. When running PEP, just make it a completely separate commit.